lib/index.js

```diff
@@ -13,7 +13,7 @@ const emptyOptions = {}
 const own = {}.hasOwnProperty
 
 /**
- * Inspect a node, without color.
+ * Inspect a tree, with color in Node, without color in browsers.
```

Comment:
```suggestion
 * Inspect a tree.
```

- Agree that `tree` is better.
- `, without color.` is of course wrong, but thinking about this again, I think the `with color in Node, without color in browsers.` is not needed? It’s about the defaults, but that isn’t mentioned.

Reply:
I think that makes a lot of sense! Here's a commit to address that: [8ef1255](https://github.com/syntax-tree/unist-util-inspect/pull/22/commits/8ef1255fbc714eb4f3b3e1a6b00f187d7206b2c1)